util/cron/test-perf.hpe-apollo-hdr.gasnet-ibv.playground.bash

```diff
@@ -25,19 +25,19 @@ export GASNET_PHYSMEM_MAX="0.90"
 #
 # When the multi-local playground is not used, set `SKIP_ML_PLAYGROUND=1
 #
+# Currently tests https://github.com/chapel-lang/chapel/pull/25517. We'll
+# determine whether to turn that optimization on-by-default.
 
 SKIP_ML_PLAYGROUND=0
 if [[ "$SKIP_ML_PLAYGROUND" == "1" ]]; then
   log_info "Skipping testing of the multi-local playground"
   exit
 fi
 
-GITHUB_USER=jhh67
-GITHUB_BRANCH=no-op
-SHORT_NAME=no-op
-START_DATE=07/08/24
-
-export CHPL_NIGHTLY_TEST_DIRS="performance/elliot"
```

Comment:
@jabraham17 I suspect this is specific to John's branch and not a must for this playground. Is that right?

Reply:
correct, that was specific to John's branch